RecoTracker/TkNavigation/test/NavigationSchoolAnalyzer.cc

```diff
@@ -62,13 +58,18 @@ class NavigationSchoolAnalyzer : public edm::EDAnalyzer {
   virtual void endJob() ;
 
   std::string theNavigationSchoolName;
+  const TrackerTopology *tTopo;
+  void print(std::ostream& os,const DetLayer* dl);
+  void print(std::ostream&os, const NavigationSchool::StateType & layers);
+  void print(std::ostream&os, const NavigationSchool *nav);
```

Comment:
Please add a space for readability to std::ostream&os on lines 63 and 64, just as you did on line 62.